plugins/woocommerce/client/blocks/tests/e2e/tests/classic-template/classic-template.block_theme.spec.ts

```diff
@@ -271,9 +272,7 @@ test.describe( `${ blockData.name } Block `, () => {
 		const editButton = editor.page.getByRole( 'menuitem', {
 			name: 'Edit',
 		} );
-
-		// Keep WP v6.7 compatibility.
-		if ( await editButton.isHidden() ) {
+		if ( wpCoreVersion >= 6.8 ) {
```

Comment:
NIT: Should we improve this logic targeting the WordPress - 1 version? In this way, we can easily refactor things when old WordPress version aren't supported anymore.